app/models/sign_in_form.rb

```diff
@@ -0,0 +1,12 @@
+class SignInForm
+  include ActiveModel::Model
+
+  attr_accessor :email, :password
```

Comment:
~ attr_accessor~ => `attr_reader`? Or are they going to overwritten at some point later?

Reply:
Changed that.  I think it can be a reader.  Can always change it if it comes up.